strawberry_django/mutations/resolvers.py

```diff
@@ -37,20 +34,20 @@
 )
 from strawberry_django.utils.inspect import get_model_fields
 
+from .dataclasses import (
```

Comment:
Maybe call this module `types.py`?

dataclasses may be conflicting, and types is IMO better because you are actually defining types there, not only dataclasses (e.g. there is a typeddict in there)

Reply:
Fair suggestions, it's a more relevant word indeed.